### ISSUE TITLE
Translate the whole screen

### ISSUE THREAD

**Issue by firemaples** [firemaples/EverTranslator#209]
Make a feature to translate all the texts on the screen at the same time with Google MLKit

**Comment by uomar8** [comment]
This is a great idea
It would be very helpful for those who use the app for

- games 
- comic
- video subtitle 

I would like to suggest that the translated text are not in a single window it should be in many windows 

Instead of this 
![Screenshot_20221231_044021](https://user-images.githubusercontent.com/7046347/210121389-c618bd5f-6852-4ae2-9d73-603f123860f0.jpg)

It could look like this
![fufucfhd (20221231043919)](https://user-images.githubusercontent.com/7046347/210121402-fbb3109b-96e4-48bc-b060-4fa6017c5461.jpg)

**Comment by firemaples** [comment]
> This is a great idea It would be very helpful for those who use the app for
> 
> * games
> * comic
> * video subtitle
> 
> I would like to suggest that the translated text are not in a single window it should be in many windows
> 
> Instead of this ![Screenshot_20221231_044021](https://user-images.githubusercontent.com/7046347/210121389-c618bd5f-6852-4ae2-9d73-603f123860f0.jpg)
> 
> It could look like this ![fufucfhd (20221231043919)](https://user-images.githubusercontent.com/7046347/210121402-fbb3109b-96e4-48bc-b060-4fa6017c5461.jpg)

Yes, that is what I'm thinking about, but the recognition quality may drop in this mode, and it will only support Latin-based, Chinese, Japanese, Korean and Devanagari languages.

Reference: https://developers.google.com/ml-kit/vision/text-recognition/v2/android

**Comment by uomar8** [comment]
In the Reference it says

-Smaller images can be processed faster.
-Consider capturing images at a lower resolution

Isn't that the opposite of the whole screen but if you have an idea wish you all the best 👍🏼
